packages/server/src/profile/profile.controller.ts

```diff
@@ -81,4 +95,39 @@ export class ProfileController {
 
     return this.get(user);
   }
+
+  @Post('/upload_picture')
+  @UseInterceptors(
+    FileInterceptor('file', {
```

Comment:
Add `limits` https://github.com/expressjs/multer#multeropts to prevent us from getting blasted by large uploads. Frontend validation is good, but backend needs to be protected

Reply:
funnily enough https://github.com/nestjs/nest/blob/f472852d441b7f66e52449e09f885ab57807bf96/packages/platform-express/multer/interfaces/multer-options.interface.ts#L15 the default is 1 MB :o. would you still want it for clarity?